flow/workflows/scheduled_flows.go

```diff
@@ -62,7 +62,8 @@ func GlobalScheduleManagerWorkflow(ctx workflow.Context) error {
 
 	slotSizeCtx := withCronOptions(ctx,
 		"record-slot-size-"+info.OriginalRunID,
-		"*/5 * * * *")
+		// TODO revert this back to 5 minutes?
```

Comment:
TODO

Reply:
On a side note @serprex should we let it be 1 minute?

Reply:
prefer to leave it same for now, changing should be separate benchmark, maybe env variable